src/configuration/tree.yaml

```diff
@@ -33,6 +33,24 @@ root:
               Scopri gli allegati tecnici delle Linee guida per lo sviluppo e la manutenzione del software
               <https://docs.italia.it/italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/stabile/index.html>
 
+          - label: "Vorrei integrare la mia piattaforma con il sistema di Single SignOn Nazionale ed Europeo"
+            content:
+
+            - label: "SPID - Sistema Pubblico di Identità Digitale"
+              content: |
+                Scopri la documentazione e tutte le risorse dedicate a SPID
+                <https://developers.italia.it/it/spid/>
```

Comment:
All the links should follow [this format](https://github.com/italia/wizard-italia/blob/main/src/configuration/tree.yaml#L20-L24), using `target="_blank"`, an `aria-label` explaining the link will lead to an external site and preferably a description with an icon signaling the external link.

Reply:
[whatch this](https://www.youtube.com/watch?v=z6jNekFYAyg)
https://github.com/italia/wizard-italia/pull/11/commits/d54ea032b675af7e20ece4861b7d0f93178c1931